package.json

```diff
@@ -9,7 +9,7 @@
     "@fontsource-variable/open-sans": "^5.0.9",
     "@formatjs/intl": "^2.0.0",
     "@giphy/js-components": "^5.0.5",
-    "@giphy/js-fetch-api": "^5.0.0",
+    "@giphy/js-fetch-api": "^5.4.0",
```

Comment:
Do Updating this require changing the provision_version in `version.py` ?

Reply:
Yes, you should bump the minor version whenever you increase a version in a way where the app won't work correctly with the old version. The comments in `PROVISION_VERSION` is intended to document this clearly, so if you find the language their confusing, please start a #backend topic about it and we can try to improve it.

Reply:
This increase is not necessary at all. We already have `@giphy/js-fetch-api@5.4.0`, which is the latest version that satisfies the semver specifier `^5.0.0`. You can see in your `pnpm-lock.yaml` diff that the version didn’t actually change, only the specifier.

```diff
   '@giphy/js-fetch-api':
-    specifier: ^5.0.0
+    specifier: ^5.4.0
     version: 5.4.0
```